python/core/symbology-ng/qgsstylev2.sip

```diff
@@ -270,6 +275,9 @@ class QgsStyleV2 : QObject
     //! gets the id from the table for the given name from the database, 0 if not found
     int getId( const QString& table, const QString& name );
 
+    //! gets the name from the table for the given id from the database, QString::null if not found
```

Comment:
@SebDieBln note that Qt discourage use of null strings - see http://doc.qt.io/qt-4.8/qstring.html#distinction-between-null-and-empty-strings . I suspect based on this that in some future Qt version null strings will be removed. It's safer to use an empty string instead.